tests/test_searcher.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from icalendar_searcher import Searcher
+
+
+def test_add_property_filter_undef():
+    """Adding an "undef" property filter should record the operator but
+    not attempt to coerce the value through TypesFactory (since value is
+    ignored for 'undef')."""
+    s = Searcher()
+    s.add_property_filter("SUMMARY", None, operator="undef")
+    assert "SUMMARY" in s._property_operator
+    assert s._property_operator["SUMMARY"] == "undef"
+    assert "SUMMARY" not in s._property_filters
+
+
+def test_add_property_filter_contains():
+    """Adding an "undef" property filter should record the operator but
+    not attempt to coerce the value through TypesFactory (since value is
+    ignored for 'undef')."""
```

Comment:
```suggestion
def test_add_property_filter_contains():
    """Adding a 'contains' property filter should record the operator
    and coerce the value through TypesFactory."""
    s = Searcher()
```

Copy-paste error. Docstring describes "undef" but test is for "contains"